staging/src/kubevirt.io/client-go/log/log.go

```diff
@@ -69,7 +69,7 @@ type LoggableObject interface {
 }
 
 type FilteredLogger struct {
-	logContext            *log.Context
```

Comment:
Does the `Context` no longer exist?

Reply:
No. Looking at the [docs](https://pkg.go.dev/github.com/go-kit/kit@v0.4.0/log) it seems like it was removed in `v0.4.0`.